music/admin.py

```diff
@@ -17,4 +27,18 @@ def add_music(self, request, queryset):
             music.room.add_music(**MusicSerializer(music).data)
         return
 
+    def duplicate_music(self,request,queryset):
+        targetRoom = Room.objects.get(name=request.POST['nrroom'])
+        if targetRoom:
+            for music in queryset:
+                if targetRoom.name != music.room.name:
+                    new_entry = music
+                    new_entry.room=targetRoom
+                    new_entry.id = None
```

Comment:
As it seems to be a hacky solution, it's still the solution mentioned by the Django doc !
With just a little observation, the Django keyword identifier is "pk" more than "id" (despite that id seems to do the job too o_O )
[Copying a model instance](https://docs.djangoproject.com/en/1.9/topics/db/queries/#copying-model-instances)
[Stackoverflow related question](http://stackoverflow.com/questions/4733609/how-do-i-clone-a-django-model-instance-object-and-save-it-to-the-database)
```
new_entry = music
new_entry.room = targetRoom
new_entry.pk = None
new_entry.save()
```